pymove/core/dataframe.py

```diff
@@ -2793,66 +2899,60 @@ def datetime(self):
 
     @property
     def loc(self):
-        # Access a group of rows and columns by label(s) or a boolean array.
+        """ Access a group of rows and columns by label(s) or a boolean array. """
+
         raise NotImplementedError("To be implemented")
 
     @property
     def iloc(self):
-        # Purely integer-location based indexing for selection by position.
+        """ Purely integer-location based indexing for selection by position. """
+
         raise NotImplementedError("To be implemented")
 
     @property
     def at(self):
-        # Access a single value for a row/column label pair.
+        """ Access a single value for a row/column label pair. """
+
         raise NotImplementedError("To be implemented")
 
     @property
     def values(self):
-        # Return a Numpy representation of the DataFrame.
+        """ Return a Numpy representation of the DataFrame. """
+
         raise NotImplementedError("To be implemented")
 
     @property
     def columns(self):
-        # The column labels of the DataFrame.
+        """ The column labels of the DataFrame. """
+
         raise NotImplementedError("To be implemented")
 
     @property
     def index(self):
-        # The index (row labels) of the DataFrame.
+        """ The index (row labels) of the DataFrame. """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should not be the function's "signature" (D402)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5536634)